istio-cni/update.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+NEW_SOURCES=""
+
+function usage() {
+    echo "Usage: $0 [-i <SHA of istio-cni>]"
+    echo
+    exit 0
+}
+
+while getopts ":i:v:" opt; do
+  case ${opt} in
+    i) ISTIO_CNI_SHA="${OPTARG}";;
+    *) usage;;
+  esac
+done
+
+[[ -z "${ISTIO_CNI_SHA}" ]] && ISTIO_CNI_SHA="$(grep '%global git_commit ' istio-cni.spec | cut -d' ' -f3)"
+
+function update_commit() {
+    local sha="$1"
+
+    local tarball="https://github.com/maistra/istio-cni/archive/${sha}/istio-cni-${sha}.tar.gz"
+    local filename="istio-cni-${sha}.tar.gz"
+
+    echo -n "Checking istio-cni...   "
+    if [ ! -f "${filename}" ]; then
+        echo "Downloading ${tarball}"
+        curl -Lfs ${tarball} -o "${filename}"
+        if [ $? -ne 0 ]; then
+            echo "Error downloading tarball, exiting."
+            exit 1
+        fi
+    else
+        echo "Already on disk, download not necessary"
+    fi
+
+    sed -i "s/%global git_commit .*/%global git_commit ${sha}/" istio-cni.spec
+    NEW_SOURCES="${NEW_SOURCES} ${filename}"
+}
+
+function new_sources() {
+    echo
+    echo "Updating sources file with ${NEW_SOURCES}"
+    md5sum ${NEW_SOURCES} > sources
```

Comment:
You don't seem to have a `sources` file...

Reply:
I do have it: https://github.com/Maistra/rpms/pull/41/files#diff-6f26da552374a0e5211923d0837f13b5

Reply:
Oh, sorry, haven't seem it. In this case, it is outdated :)

Reply:
sorry, need a coffee